### ISSUE TITLE
Harmonize epub vocabulary definitions with ARIA roles

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#533]
A number of definitions have been cleaned up during the port to ARIA to rely less on implied publishing expectations of their use.

To avoid future confusion, I'd recommend updating all epub definitions to match the ARIA definitions at:
https://rawgit.com/w3c/aria/master/aria/dpub.html

**Comment by TzviyaSiegman** [comment]
+1 @mattgarrish is this something that you can easily script?

**Comment by laudrain** [comment]
I'm afraid some entries in SSV are not in ARIA role definition, like : 
- document divisions : volume and division
- document navigation : landmarks and lists
- index : all 'index-*' 
- several preliminaries
  Is there a plan to improve ARIA vocab?

**Comment by mattgarrish** [comment]
The vocabulary might be expanded in the future, but the goal for the first draft was to provide a minimal set of core roles.

It is possible to identify other sections using generic ARIA roles, for example:

``` html
<section role="region" aria-labelledby="vol">
   <h1 id="vol">Volume 1</h1>
```

**Comment by mattgarrish** [comment]
@TzviyaSiegman We don't need to do anything for this one. I updated all our definitions to match prior to the last release. This issue was opened at the start of the revision as a placeholder so we didn't forget about the DPUB ARIA vocab work. We can probably just close it off now, as I expect we're done with tweaking the ARIA definitions.

**Comment by laudrain** [comment]
I was wondering how to script it if there is no corresponding terms?

**Comment by mattgarrish** [comment]
I could write something if we really need. I'd just have to parse the terms/definitions from the w3c page into a hashtable then use that to run through the epub ssv n3 file, ignoring terms that aren't present.

But we really just opened this as a placeholder as we knew we needed to update the ssv, but weren't sure when we'd be done updating the aria roles. Now that we've updated, future changes should be minimal.
